shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -329,12 +329,16 @@ - (void)touchesCancelled:(NSSet*)touches withEvent:(UIEvent*)event {
 #pragma mark - Handle view resizing
 
 - (void)updateViewportMetrics {
+  // Update viewport metrics for the engine.
   blink::Threads::UI()->PostTask([
     engine = _platformView->engine().GetWeakPtr(), metrics = _viewportMetrics
   ] {
     if (engine.get())
       engine->SetViewportMetrics(metrics);
   });
+
+  // Tell the platform view that its surface was updated.
+  _platformView->SurfaceSizeChanged();
```

Comment:
This creates a race condition where the Platform thread will notify both the UI and GPU threads of this change but there won't be any ordering relationship between those threads receiving this notification.  Maybe have the UI thread notify the GPU thread or sychronize the updates?